code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -37,9 +37,10 @@
 	/// List of available condibottle styles for UI
 	var/list/condi_styles
 
-/obj/machinery/chem_master/Initialize()
-	create_reagents(100)
+	var/datum/player_reagent_list/reagent_guide
```

Comment:
Consider turning this into a `static` if every chem master shares the guide, that way we won't have to generate it for every instance.

Reply:
> Consider turning this into a `static`/`global`[1] if every chem master shares the guide, that way we won't have to generate it for every instance.
> 
> [1] https://www.byond.com/docs/ref/#/var/global (https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-static-instead-of-global)

Current information uses static but maybe it works better with this? Learning time I suppose